x-pack/plugins/security_solution/server/endpoint/services/fleet/endpoint_fleet_services_factory.ts

```diff
@@ -82,21 +82,20 @@ export interface EndpointFleetServicesInterface {
   packagePolicy: PackagePolicyClient;
 }
 
-export interface EndpointScopedFleetServicesInterface extends EndpointFleetServicesInterface {
+export interface EndpointInternalFleetServicesInterface extends EndpointFleetServicesInterface {
```

Comment:
so this turned out to be a typo?

Reply:
Yeah, but also some of the class methods needed changes after the interfaces were correctly re-named and the types were changed.